pkg/limayaml/default.yaml

```diff
@@ -167,7 +167,8 @@ network:
 
 # # extra environment variables that will be loaded into the VM at start up
 # env:
-#   KEY: value
+#   - key: key1
+#     value: value
```

Comment:
I’d prefer to keep the previous YAML syntax

Reply:
> I’d prefer to keep the previous YAML syntax

Yes, me too. The list form only makes sense when you can legitimately have multiple entries with the same key. But the environment is a map/hash, not a slice/array.